contents/tutorials/supabase-query.md

```diff
@@ -0,0 +1,166 @@
+---
+title: How to query Supabase in PostHog
+date: 2024-08-01
+author:
+ - ian-vanagas
+tags:
+ - data warehouse
+---
+
+Combining your app and analytics data is a powerful way to understand your product. Supabase is a popular choice for handling that app data, as it provides a database, auth, storage, and more all-in-one.
```

Comment:
I think "database" sounds more right, even though supabase is technically more things. When I first read "app data", it wasnt clear to me what it was 
```suggestion
Combining your database and analytics data is a powerful way to understand your users. [Supabase](https://supabase.com/) is a popular choice for handling that app data, as it provides a database, auth, storage, and more all-in-one.
```